apps/cookbook/src/app/app.routes.ts

```diff
@@ -69,11 +69,6 @@ export const routes: Routes = [
   },
   {
     path: 'examples',
-    loadChildren: () =>
-      import('./examples/examples-routing.module').then((m) => m.ExamplesRoutingModule),
-  },
-  {
-    path: 'designer',
-    loadChildren: () => import('./designer/designer.module').then((m) => m.DesignerModule),
```

Comment:
The designer component is unused legacy code, and I have removed it all together